{
  "uid" : "8d815137d02db86f",
  "name" : "Verify the composition and visibility of elements \n        on the 1st-6th levels of nesting on the page",
  "fullName" : "tests.specialists_page_test.TestSpecialistsPageStructure#test_sp_01_02_verify_page_structure_and_visibility",
  "historyId" : "c1d43c00f2db04da242760b37326b756",
  "time" : {
    "start" : 1728784202131,
    "stop" : 1728784202131,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570d05d702a <unknown>\n#1 0x5570d02bd5e0 <unknown>\n#2 0x5570d02b57eb <unknown>\n#3 0x5570d02a5a09 <unknown>\n#4 0x5570d02a7744 <unknown>\n#5 0x5570d02a5cbd <unknown>\n#6 0x5570d02a552f <unknown>\n#7 0x5570d02a5422 <unknown>\n#8 0x5570d02a3493 <unknown>\n#9 0x5570d02a3ada <unknown>\n#10 0x5570d02bff41 <unknown>\n#11 0x5570d03518c5 <unknown>\n#12 0x5570d0331b22 <unknown>\n#13 0x5570d0350d7d <unknown>\n#14 0x5570d03318c3 <unknown>\n#15 0x5570d02ff6b3 <unknown>\n#16 0x5570d030068e <unknown>\n#17 0x5570d05a1a2b <unknown>\n#18 0x5570d05a59b1 <unknown>\n#19 0x5570d058e225 <unknown>\n#20 0x5570d05a6532 <unknown>\n#21 0x5570d057338f <unknown>\n#22 0x5570d05c5f28 <unknown>\n#23 0x5570d05c60f3 <unknown>\n#24 0x5570d05d5e7c <unknown>\n#25 0x7f4874c94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"df4658870716c009058558ab83005584\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f6f0800>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5570d05c60f3 \\\\u003Cunknown>\\\\n#24 0x5570d05d5e7c \\\\u003Cunknown>\\\\n#25 0x7f4874c94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5570d05d702a <unknown>\nE       #1 0x5570d02bd5e0 <unknown>\nE       #2 0x5570d02b57eb <unknown>\nE       #3 0x5570d02a5a09 <unknown>\nE       #4 0x5570d02a7744 <unknown>\nE       #5 0x5570d02a5cbd <unknown>\nE       #6 0x5570d02a552f <unknown>\nE       #7 0x5570d02a5422 <unknown>\nE       #8 0x5570d02a3493 <unknown>\nE       #9 0x5570d02a3ada <unknown>\nE       #10 0x5570d02bff41 <unknown>\nE       #11 0x5570d03518c5 <unknown>\nE       #12 0x5570d0331b22 <unknown>\nE       #13 0x5570d0350d7d <unknown>\nE       #14 0x5570d03318c3 <unknown>\nE       #15 0x5570d02ff6b3 <unknown>\nE       #16 0x5570d030068e <unknown>\nE       #17 0x5570d05a1a2b <unknown>\nE       #18 0x5570d05a59b1 <unknown>\nE       #19 0x5570d058e225 <unknown>\nE       #20 0x5570d05a6532 <unknown>\nE       #21 0x5570d057338f <unknown>\nE       #22 0x5570d05c5f28 <unknown>\nE       #23 0x5570d05c60f3 <unknown>\nE       #24 0x5570d05d5e7c <unknown>\nE       #25 0x7f4874c94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728784202131,
      "stop" : 1728784202915,
      "duration" : 784
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728784202915,
      "stop" : 1728784203340,
      "duration" : 425
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570d05d702a <unknown>\n#1 0x5570d02bd5e0 <unknown>\n#2 0x5570d02b57eb <unknown>\n#3 0x5570d02a5a09 <unknown>\n#4 0x5570d02a7744 <unknown>\n#5 0x5570d02a5cbd <unknown>\n#6 0x5570d02a552f <unknown>\n#7 0x5570d02a5422 <unknown>\n#8 0x5570d02a3493 <unknown>\n#9 0x5570d02a3ada <unknown>\n#10 0x5570d02bff41 <unknown>\n#11 0x5570d03518c5 <unknown>\n#12 0x5570d0331b22 <unknown>\n#13 0x5570d0350d7d <unknown>\n#14 0x5570d03318c3 <unknown>\n#15 0x5570d02ff6b3 <unknown>\n#16 0x5570d030068e <unknown>\n#17 0x5570d05a1a2b <unknown>\n#18 0x5570d05a59b1 <unknown>\n#19 0x5570d058e225 <unknown>\n#20 0x5570d05a6532 <unknown>\n#21 0x5570d057338f <unknown>\n#22 0x5570d05c5f28 <unknown>\n#23 0x5570d05c60f3 <unknown>\n#24 0x5570d05d5e7c <unknown>\n#25 0x7f4874c94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728784202915,
        "stop" : 1728784203340,
        "duration" : 425
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5570d05d702a <unknown>\n#1 0x5570d02bd5e0 <unknown>\n#2 0x5570d02b57eb <unknown>\n#3 0x5570d02a5a09 <unknown>\n#4 0x5570d02a7744 <unknown>\n#5 0x5570d02a5cbd <unknown>\n#6 0x5570d02a552f <unknown>\n#7 0x5570d02a5422 <unknown>\n#8 0x5570d02a3493 <unknown>\n#9 0x5570d02a3ada <unknown>\n#10 0x5570d02bff41 <unknown>\n#11 0x5570d03518c5 <unknown>\n#12 0x5570d0331b22 <unknown>\n#13 0x5570d0350d7d <unknown>\n#14 0x5570d03318c3 <unknown>\n#15 0x5570d02ff6b3 <unknown>\n#16 0x5570d030068e <unknown>\n#17 0x5570d05a1a2b <unknown>\n#18 0x5570d05a59b1 <unknown>\n#19 0x5570d058e225 <unknown>\n#20 0x5570d05a6532 <unknown>\n#21 0x5570d057338f <unknown>\n#22 0x5570d05c5f28 <unknown>\n#23 0x5570d05c60f3 <unknown>\n#24 0x5570d05d5e7c <unknown>\n#25 0x7f4874c94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"df4658870716c009058558ab83005584\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784203373,
      "stop" : 1728784203435,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 27,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "9e07faf29a0e9b02",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/9e07faf29a0e9b02",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f2bc72e02a <unknown>\n#1 0x55f2bc4145e0 <unknown>\n#2 0x55f2bc40c7eb <unknown>\n#3 0x55f2bc3fca09 <unknown>\n#4 0x55f2bc3fe744 <unknown>\n#5 0x55f2bc3fccbd <unknown>\n#6 0x55f2bc3fc52f <unknown>\n#7 0x55f2bc3fc422 <unknown>\n#8 0x55f2bc3fa493 <unknown>\n#9 0x55f2bc3faada <unknown>\n#10 0x55f2bc416f41 <unknown>\n#11 0x55f2bc4a88c5 <unknown>\n#12 0x55f2bc488b22 <unknown>\n#13 0x55f2bc4a7d7d <unknown>\n#14 0x55f2bc4888c3 <unknown>\n#15 0x55f2bc4566b3 <unknown>\n#16 0x55f2bc45768e <unknown>\n#17 0x55f2bc6f8a2b <unknown>\n#18 0x55f2bc6fc9b1 <unknown>\n#19 0x55f2bc6e5225 <unknown>\n#20 0x55f2bc6fd532 <unknown>\n#21 0x55f2bc6ca38f <unknown>\n#22 0x55f2bc71cf28 <unknown>\n#23 0x55f2bc71d0f3 <unknown>\n#24 0x55f2bc72ce7c <unknown>\n#25 0x7f4ae8694ac3 <unknown>",
        "time" : {
          "start" : 1728697316051,
          "stop" : 1728697316051,
          "duration" : 0
        }
      }, {
        "uid" : "9a180ee5986de8af",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/9a180ee5986de8af",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5631f1a5d02a <unknown>\n#1 0x5631f17435e0 <unknown>\n#2 0x5631f173b7eb <unknown>\n#3 0x5631f172ba09 <unknown>\n#4 0x5631f172d744 <unknown>\n#5 0x5631f172bcbd <unknown>\n#6 0x5631f172b52f <unknown>\n#7 0x5631f172b422 <unknown>\n#8 0x5631f1729493 <unknown>\n#9 0x5631f1729ada <unknown>\n#10 0x5631f1745f41 <unknown>\n#11 0x5631f17d78c5 <unknown>\n#12 0x5631f17b7b22 <unknown>\n#13 0x5631f17d6d7d <unknown>\n#14 0x5631f17b78c3 <unknown>\n#15 0x5631f17856b3 <unknown>\n#16 0x5631f178668e <unknown>\n#17 0x5631f1a27a2b <unknown>\n#18 0x5631f1a2b9b1 <unknown>\n#19 0x5631f1a14225 <unknown>\n#20 0x5631f1a2c532 <unknown>\n#21 0x5631f19f938f <unknown>\n#22 0x5631f1a4bf28 <unknown>\n#23 0x5631f1a4c0f3 <unknown>\n#24 0x5631f1a5be7c <unknown>\n#25 0x7f1f47294ac3 <unknown>",
        "time" : {
          "start" : 1728611007284,
          "stop" : 1728611007284,
          "duration" : 0
        }
      }, {
        "uid" : "9105dee3f5933d8c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/9105dee3f5933d8c",
        "status" : "passed",
        "time" : {
          "start" : 1728524973571,
          "stop" : 1728524973907,
          "duration" : 336
        }
      }, {
        "uid" : "dc15f54dd7375bca",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/dc15f54dd7375bca",
        "status" : "passed",
        "time" : {
          "start" : 1728438645505,
          "stop" : 1728438645850,
          "duration" : 345
        }
      }, {
        "uid" : "b6d4dfb6c8eeb1a9",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/b6d4dfb6c8eeb1a9",
        "status" : "passed",
        "time" : {
          "start" : 1728352228687,
          "stop" : 1728352229045,
          "duration" : 358
        }
      }, {
        "uid" : "a9094eb42bc7e03f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/a9094eb42bc7e03f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find elements by locator ('xpath', '//main/*/*/*/*/*/*')",
        "time" : {
          "start" : 1728266641777,
          "stop" : 1728266662232,
          "duration" : 20455
        }
      }, {
        "uid" : "1a27514a0a1e5c80",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/1a27514a0a1e5c80",
        "status" : "passed",
        "time" : {
          "start" : 1728179773071,
          "stop" : 1728179773416,
          "duration" : 345
        }
      }, {
        "uid" : "f5c590643dda17ec",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/f5c590643dda17ec",
        "status" : "passed",
        "time" : {
          "start" : 1728093009480,
          "stop" : 1728093009827,
          "duration" : 347
        }
      }, {
        "uid" : "d6fe76554e86c81f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/d6fe76554e86c81f",
        "status" : "passed",
        "time" : {
          "start" : 1728006712647,
          "stop" : 1728006713001,
          "duration" : 354
        }
      }, {
        "uid" : "fd7d281bfb2b7d17",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/fd7d281bfb2b7d17",
        "status" : "passed",
        "time" : {
          "start" : 1727920356286,
          "stop" : 1727920356663,
          "duration" : 377
        }
      }, {
        "uid" : "6b523dbc482ce4aa",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/6b523dbc482ce4aa",
        "status" : "passed",
        "time" : {
          "start" : 1727833782964,
          "stop" : 1727833783313,
          "duration" : 349
        }
      }, {
        "uid" : "f4e83c421f4fbd7a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/f4e83c421f4fbd7a",
        "status" : "passed",
        "time" : {
          "start" : 1727747907612,
          "stop" : 1727747907974,
          "duration" : 362
        }
      }, {
        "uid" : "f1030db456066888",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/f1030db456066888",
        "status" : "passed",
        "time" : {
          "start" : 1727661189708,
          "stop" : 1727661190051,
          "duration" : 343
        }
      }, {
        "uid" : "96deca430122a7a5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/96deca430122a7a5",
        "status" : "passed",
        "time" : {
          "start" : 1727575010755,
          "stop" : 1727575011098,
          "duration" : 343
        }
      }, {
        "uid" : "52875ebbbf38804f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/52875ebbbf38804f",
        "status" : "passed",
        "time" : {
          "start" : 1727488236990,
          "stop" : 1727488237330,
          "duration" : 340
        }
      }, {
        "uid" : "77daa9d92a691027",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/77daa9d92a691027",
        "status" : "passed",
        "time" : {
          "start" : 1727401978106,
          "stop" : 1727401978448,
          "duration" : 342
        }
      }, {
        "uid" : "a17b68ee3bfb90e3",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/a17b68ee3bfb90e3",
        "status" : "passed",
        "time" : {
          "start" : 1727315452970,
          "stop" : 1727315453339,
          "duration" : 369
        }
      }, {
        "uid" : "46eaf2915b9d39a",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/46eaf2915b9d39a",
        "status" : "passed",
        "time" : {
          "start" : 1727229104535,
          "stop" : 1727229104893,
          "duration" : 358
        }
      }, {
        "uid" : "6bc355b26b75271f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/6bc355b26b75271f",
        "status" : "passed",
        "time" : {
          "start" : 1727142635761,
          "stop" : 1727142636135,
          "duration" : 374
        }
      }, {
        "uid" : "ddef590d4aa0a5c1",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/ddef590d4aa0a5c1",
        "status" : "passed",
        "time" : {
          "start" : 1727056157257,
          "stop" : 1727056157594,
          "duration" : 337
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "8d815137d02db86f.json",
  "parameterValues" : [ ]
}